recipes/pyactivestorage/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set python_min = "3.10" %}
+{% set name = "PyActiveStorage" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: 8931b55898be7b8e317bc3264d3396005330aa1e5a24110dfb675daa83a3ea93
+
+build:
+  number: 0
+  # not 100% noarch; middleware is not-Win so not a problem
+  # only test failing for Win is tests/unit/test_mock_s3.py
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip !=21.3
+    - pytest-runner
+    - setuptools >=40.6.0
+    - setuptools_scm >=6.2
+  run:
+    - python >={{ python_min }}
+    - pyfive >=0.5.0  # earliest support for advanced Pyfive
```

Comment:
Many of the comments here seems to be upstream issues and/or notes for the future. Maybe drop them and open issues when the feedstock is created? They kind of pollute the recipe are likely to be ignored when merging bot PRs.